### ISSUE TITLE
Run frontend and backend servers in parallel

### ISSUE THREAD

**Issue by DTFsquad** [DTFsquad/Kodiri-Kodflix-MERN#21]
We still need two commands to startup frontend and backend servers:

- frontend: npm start.
- backend: npm start-backend.
We want to start both servers using a single command using npm.

Technical guidance:

1. At the moment, both servers are running on the same URL (on port 3000). We have to change one of the ports as we cannot have multiple servers running on the same port at the same time. Hence, replace the port on app.js, from 3000 to 3001.

2. Install npm-run-all. Note the changes on package.json

3. Create a new task start-all on package.json to run both npm tasks in parallel: npm-run-all -p <task1> <task2>.

```
...
"scripts": {
    ...
    "start-all": "npm-run-all..."
    ...
"}"
...
```
4. Prove both servers work:
http://localhost:3000 -> We get the frontend server, with the TV show covers
http://localhost:3001 -> We get the backend server, with the TV show data
Please note, at this stage, both servers are still disconnected. That means, data is not coming to the frontend from the backend. But that's fine, will connect both servers shortly.

Original source:
https://github.com/rmallols/kodflix/issues/21
